### DISCUSSION TITLE
Should I commit files that changed after running ./gradle jApiCmp

### DISCUSSION THREAD

**Discussion by hongshuai1994** [open-telemetry/opentelemetry-java#6561] [Q&A] [answered]
I am trying to submit a PR, but some Checks have not passed. Check the Checks error content. The prompts are as follows.
![image](https://github.com/git-ecosystem/git-credential-manager/assets/80237674/0f807d52-1bc6-4703-9ae7-75b4f5a12348)
I ran .`/gradlew jApiCmp`, but file modifications occurred. Should I submit these files to the PR?
![image](https://github.com/git-ecosystem/git-credential-manager/assets/80237674/acedd7a4-2dce-4b1c-a28a-032f29333e8f)

**Comment by jkwatson** [comment, marked as answer]
yes, those need to be submitted as a part of PRs, normally. However, we're in the middle of getting our post-release automation straightened out, which should get the diff docs up to date. So, unless you're changing the public APIs, it _shouldn't_ be necessary to include them in the PR.
